### ISSUE TITLE
Wrong syntax highlight with `context` or `describe` and escaped quote

### ISSUE THREAD

**Issue by shir** [pauldruziak/language-rspec#42]
Example code:
```ruby
require 'spec_helper'

describe 'test\'s' do
   ...
end
```

Within this code syntax highlighter interpret `\'` as end of string:

<img width="242" alt="screen shot 2017-03-20 at 14 15 48" src="https://cloud.githubusercontent.com/assets/229417/24097619/c1bd44f6-0d77-11e7-88e6-1abbf8a17ed2.png">

Issue appear only for `describe` and `context`. For `it` it works as expected.

**Comment by ryansobol** [comment]
@shir Thanks for reporting the bug. A fix is incoming in v0.9.0.

**Comment by ryansobol** [comment]
Also, the change contains a fix for the following code snippet.

```ruby
describe '\\' do
  #
end
```
